teuthology/beanstalk.py

```diff
@@ -1,11 +1,15 @@
 from __future__ import print_function
-import beanstalkc
-import yaml
 import logging
 import pprint
 import sys
 from collections import OrderedDict
 
+import yaml
+try:
+    import beanstalkc
+except ImportError:
+    import pystalkd.Beanstalkd as beanstalkc
+
 from .config import config
 from . import report
```

Comment:
did you try various teuthology-queue operations? Wondering if there's any more adjustment needed for pystalkd